alignak/satellite.py

```diff
@@ -691,33 +759,35 @@ def do_get_new_actions(self):
                 continue
 
             try:
-                try:
-                    con = sched['con']
-                except KeyError:  # pragma: no cover, simple protection
-                    con = None
-                if con is not None:  # None = not initialized
+                con = sched.get('con', None)
+                if con is None:  # None = not initialized
+                    self.pynag_con_init(sched_id)
+                    sched['con'] = con
+
+                if con:
```

Comment:
Hmm... you found out a bug 😉 
It should be: `con = sched['con']` ... I did not catched this one!